README.md

```diff
@@ -1 +1,12 @@
 # ELITE TEAM PERSONAL SKILLS
+
+## Mansoor Mustapha 0002
+
+### **General Skills**
+
+1. [Communication Skills](General_Skills_0002/Communication_Skills_%231_0002.md)
```

Comment:
Reconsider the naming convention used while referring to Communication Skills files.

Reply:
Kindly, check my file in [https://github.com/usanaphtal112/elite-git-practice-v2/tree/ch-Mansoor-0002/General_Skills_0002]
I used the recommended naming convention. I just made a slight change in Links not a file. You can check my first commit I used the recommended style but it didn't function ( leading to 404 errror)

Reply:
Ok, here is the reason, "As in Markdown # symbol is used to create the heading, it is considered as the special character which is not allowing it to be used as the naming character, in that way it can be is represented as "**%23**".